pkg/serverutils/utils.go

```diff
@@ -25,6 +26,28 @@ func SendResponse(rw http.ResponseWriter, code int, resp interface{}) {
 	}
 }
 
+func CheckUnsupportedBrowser(r *http.Request) bool {
+	userAgentHeader := r.Header.Get("User-Agent")
+	isUnsupported := false
+	unsupportedBrowserHeadersIdentifier := []string{
+		"Trident/", // IE 11
+		"MSIE ",    // IE 10 or older
+	}
+	for _, identifier := range unsupportedBrowserHeadersIdentifier {
+		if strings.Contains(userAgentHeader, identifier) {
+			isUnsupported = true
+			break
+		}
+	}
+	return isUnsupported
+}
+
+func SendUnsupportedBrowserResponse(rw http.ResponseWriter) {
+	rw.Header().Set("Content-Type", "text/html")
+	rw.WriteHeader(http.StatusFailedDependency)
+	rw.Write([]byte("<div>Unsupported browser detected. Please use a supported browser such as Google Chrome, Mozilla Firefox, or Microsoft Edge.</div>"))
```

Comment:
Safari is also supported. I would add some more detail here with a link to the tested integrations. Maybe

```html
<p>Internet Explorer 11 and earlier are not supported, and the {brand} console will not function with your current browser.
Please use a supported browser such as the latest version of Google Chrome, Mozilla Firefox, Microsoft Edge, or Safari.
The current Firefox ESR (Extended Support Release) is also supported. To see the browsers we test, refer to the
<a href="https://access.redhat.com/articles/4763741">OpenShift Container Platform 4.x Tested Integrations (for x86_x64)</a>.
```

Reply:
It might be good to link to the download pages for the various browsers as well.

Reply:
I'm not sure I'd mention Safari given it is no longer available for Windows (which anyone seeing this message will be using since they're using IE11).

I favor the message as it is for its brevity and maintainability (there is little that can become outdated).  Very few users should be seeing this message.  When they do, the important part  to communicate is the browser they are using is not supported.  As has been discussed in Slack, we need better documentation on supported browsers, but putting more of that here seems like overkill.  Maybe we link to that doc when it exists?